schema.sql

```diff
@@ -44,3 +44,29 @@ ALTER TABLE animals
   ADD CONSTRAINT fk_owner
   FOREIGN KEY (owner_id)
   REFERENCES owners (id);
+
+-- Creating new table vets
+CREATE TABLE vets (
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(100) NOT NULL, 
+  age INT,
+  date_of_graduation DATE NOT NULL 
+)
+
+--Specialization table
+
+CREATE TABLE specializations (
+    species_id INT NOT NULL,
+    vet_id INT NOT NULL,
+    FOREIGN KEY (species_id) REFERENCES species (id),
+    FOREIGN KEY (vet_id) REFERENCES vets (id) 
+);
+
+--Visit table
+CREATE TABLE visits (
+    animal_id INT NOT NULL,
+    vet_id INT NOT NULL,
+    date_of_visit DATE,
+    FOREIGN KEY (animal_id) REFERENCES animals (id),
+    FOREIGN KEY (vet_id) REFERENCES vets (id) 
+);
```

Comment:
## Your code 
```sql
CREATE TABLE owners (
    id SERIAL PRIMARY KEY NOT NULL,
    full_name varchar(100),
    age INT,
    PRIMARY KEY(owner_id)
);
```
- [ ] Kindly use one primary key I am getting an error when trying to create an owner table.
![image](https://user-images.githubusercontent.com/65068771/154518320-f9e008f9-0d0e-4e04-b529-9670c6c0f0cf.png),